database/migrations/2018_06_04_128200_create_sub_applications_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateSubApplicationsTable extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('sub_applications', function (Blueprint $table) {
+            $table->increments('id');
+            $table->uuid('uid')->unique();
+            $table->integer('application_id')->nullable()->unsigned();
```

Comment:
Should this be nullable? Seems the application_id is required to create a sub_application (it is a sub after all)